tentacle/src/transports/mod.rs

```diff
@@ -162,7 +166,7 @@ mod os {
 
         fn listen(self, address: Multiaddr) -> Result<Self::ListenFuture> {
             match find_type(&address) {
-                TransportType::Tcp => {
+                TransportType::Tcp | TransportType::Onion => {
```

Comment:
onion should return `Err(TransportErrorKind::NotSupported(address))`？

Reply:
Thank you, Fixed.